server/src/main/java/org/elasticsearch/common/LocalTimeOffset.java

```diff
@@ -91,7 +91,7 @@ public static Lookup lookup(ZoneId zone, long minUtcMillis, long maxUtcMillis) {
      *
      * @return a lookup function of {@code null} if none could be built 
      */
-    public static LocalTimeOffset lookupFixedOffset(ZoneId zone) {
+    public static LocalTimeOffset fixedOffset(ZoneId zone) {
```

Comment:
I renamed this method because the name seemed wrong to me when I read it this time around.